webpack.config.js

```diff
@@ -1,11 +1,16 @@
-module.exports = {
+/*! Copyright 2021 Ayogo Health Inc. */
+
+const path = require('path');
+const ContextReplacementPlugin = require('webpack/lib/ContextReplacementPlugin');
+const ModuleConcatenationPlugin = require('webpack/lib/optimize/ModuleConcatenationPlugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const CSSGeorge = require('less-plugin-css-george');
```

Comment:
I don't think this component has any CSS variables, so we probably don't need to pull this in